tests/kafkatest/tests/streams/streams_upgrade_test.py

```diff
@@ -349,56 +370,42 @@ def get_version_string(self, version):
     def start_all_nodes_with(self, version):
```

Comment:
I refactored this method to start all the nodes concurrently, rather than one at a time. We still do a rolling upgrade, but there's no need to do a rolling startup.